### PR TITLE
[master < ] Add upload to S3 job to Package All workflow

### PR DESCRIPTION
**Pull request by brunos252**
- introduced a version input for two purposes, to automatically upload to that folder, and to use as a condition if we don't want to upload artifacts, just run package_all as it was
- renamed two artifact names from `arm` to `aarch` because, by not specifying artifacts to download (comment in code), they are all downloaded and placed in folder of same name, so it is easy to just upload that folder to S3. [Reference](https://github.com/actions/download-artifact#download-all-artifacts)

### CODE REVIEW

**Comment by brunos252** [comment]
The second file, upload-to-s3.yaml is an alternative solution, workflow which uses artifacts by a previous package_all.yaml workflow. It uses a third party workflow I found here: https://stackoverflow.com/questions/60355925/share-artifacts-between-workflows-github-actions

**Comment by tonilastre** [comment]
@brunos252 the action `jakejarvis/s3-sync-actions` will create any missing subpath you define in the workflow when uploading artifacts to the S3. I am not sure about the other action that is used in that other workflow.

**Review by jbajic — COMMENTED** [review]
I think you can also test it out just to make sure that the workflow works using GithubCli, you can run workflows that are not on master

File: .github/workflows/package_all.yaml [hunk 1]
```diff
@@ -225,5 +230,30 @@ jobs:
...
         uses: actions/upload-artifact@v3
         with:
-          name: ubuntu-22.04-arm
+          name: ubuntu-22.04-aarch64
           path: build/output/ubuntu-22.04-arm/memgraph*.deb
+
+  upload-to-s3:
+    # only run upload if we specified version. Allows for runs without upload
+    if: "${{ github.event.inputs.memgraph_version != '' }}"
+    needs: [centos-7, centos-9, debian-10, debian-11, docker, ubuntu-1804, ubuntu-2004, ubuntu-2204, debian-11-platform, fedora-36, amzn-2, debian-11-arm, ubuntu-2204-arm]
```
[the remark this answers is not in the document]
Comment (jbajic) [reply .github/workflows/package_all.yaml]: Yes you are right

**Comment by brunos252** [comment]
Test run of upload workflow:
https://github.com/memgraph/memgraph/actions/runs/4608638800

**Pull request merged**